frog/imports/ui/GraphEditor/SidePanel/fileConvert.jsx

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import React, { Component } from 'react';
+import Form from 'react-jsonschema-form';
+import CopyToClipboard from 'react-copy-to-clipboard';
+
+import { Uploads } from '/imports/api/uploads';
+
+class fileConvert extends Component {
+  state: { url: string, copied: boolean, display: boolean };
+
+  constructor(props: {}) {
+    super(props);
+    this.state = { url: '', copied: false, display: false };
```

Comment:
Do you need the field `copied` in the state? I don't see how the component changes depending on that property.